tests/bin/install.sh

```diff
@@ -123,6 +123,69 @@ install_db() {
 	mysqladmin create $DB_NAME --user="$DB_USER" --password="$DB_PASS"$EXTRA
 }
 
+install_e2e_site() {
```

Comment:
Maybe for a future iteration, we can explore using the same WP install that is used for unit tests and just replace the database? This should be faster than installing another WP instance and I don't see any drawbacks, but I'm not sure if it is possible. `install_e2e_site()` is adding about 40 seconds to `install.sh` execution time (https://travis-ci.org/woocommerce/woocommerce/jobs/329686529#L518). When `install_e2e_site()` is not called, `install.sh` takes about 10 seconds to run.

I don't see this as a blocker, and I'm happy to merge this as is this week since next week we have the 3.3 release.

Reply:
I like that idea. Would be good to look in to for future iterations